x/tx/CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+### Features
```

Comment:
Shall prepare x/tx v0.13.2 now or do we have anything related to protov2 / gogo to get in before?
cc @kocubinski